doc/api/os.md

```diff
@@ -33,205 +44,294 @@ defined are described in [OS Constants][].
 added: v0.3.3
 -->
 
-Returns an array of objects containing information about each CPU/core
-installed: model, speed (in MHz), and times (an object containing the number of
-milliseconds the CPU/core spent in: user, nice, sys, idle, and irq).
+The `os.cpus()` method returns an array of objects containing information about
+each CPU/core installed.
 
-Example inspection of os.cpus:
+The properties included on each object include:
+
+* `model` {String}
+* `speed` {number} (in MHz)
```

Comment:
I'm planning to make a second pass to improve the consistency of argument types. For now I'm focusing primarily on making sure that arguments are even *listed* in the first place.